conf/script/v2/create-cluster

```diff
@@ -0,0 +1,49 @@
+#!/bin/python3
```

Comment:
@azheregelya 
Here in name 
```
conf/script/v2/create-cluster
```
i see it is python file
can we use .py extension 

We need all python file only in cortx-ha/ha/ dir

Reply:
Renamed, moved to ha/setup/v2
Actually I'm not sure that v2 is needed, so it can be deleted.

Reply:
v2 directory is deleted. files now dwell in ha/setup